include/boost/geometry/arithmetic/determinant.hpp

```diff
@@ -39,7 +39,13 @@ public :
     static inline ReturnType apply(U const& ux, U const& uy
                                  , V const& vx, V const& vy)
     {
-        return rt(ux) * rt(vy) - rt(uy) * rt(vx);
```

Comment:
Have you tried writing this as:
```
    return (rt(ux) * rt(vy)) - (rt(uy) * rt(vx));
```
It should give the same results as your alternative code.